### DISCUSSION TITLE
Sobrecarga de funções

### DISCUSSION THREAD

**Discussion by MelAndBug** [dgadelha/Portugol-Webstudio#167] [Ideias e sugestões]
Olá acho que seria uma boa idea implementar a sobrecarga de funções.
Hojé se definimos duas funções com o mesmo nome e tipo de dado de retorno diferente  o programa retorna função já declarada.

![image](https://github.com/dgadelha/Portugol-Webstudio/assets/110874840/f892c783-faec-4e01-9768-6f077dc8df10)
![image](https://github.com/dgadelha/Portugol-Webstudio/assets/110874840/78b845ad-5b05-4240-84b4-0539055282fa)

**Comment by dgadelha** [comment]
Olá! Estou avaliando ainda a possibilidade de o Portugol Webstudio se "distanciar" do Portugol Studio, incrementando a linguagem.

Imagino que isso só deva acontecer após 100% dos recursos do Portugol Studio serem suportados (bibliotecas ausentes, execução passo a passo).

Por ora então, a solicitação de mudanças na linguagem deve ser feita em https://github.com/UNIVALI-LITE/Portugol-Studio
